apps/condo/access.js

```diff
@@ -3,7 +3,7 @@ const { get } = require('lodash')
 
 const { getByCondition, getById } = require('@core/keystone/schema')
 const { userIsAuthenticated, userIsAdmin } = require('@core/keystone/access')
-const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
```

Comment:
It's better to keep it inside the utils. Access exports only access check methods and not utilities

Reply:
Moved back to organization/utils/accessSchema